### PR TITLE
version and preliminary changelog

### PR DESCRIPTION
**Pull request by Bob-Murphy**
preliminary changelog and version update for 3.6.0

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -3,6 +3,96 @@ Changelog
...
+   * :issue:`` 
+   * :issue:`` 
+
+.. admonition:: Template updates
+
+   * :issue:`3203` Added Poseidon II to RHS NAPA
+   * :issue:`3204` Added Reaction Forces CDLC compatibility for Western Sahara templates
+   * :issue:`3208` Added vehicles for vehiclesAirPatrol to some RHS templates
+   * :issue:`3258` "Re-added" the RPD to the 3CB SOV template
+   * :issue:``
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply changelog.rst:38]: Already mentioned in line 47. Both makes no sense.

File: changelog.rst [hunk 2]
```diff
@@ -3,6 +3,118 @@ Changelog
...
+   * :issue:`` 
+   * :issue:`` 
+   * :issue:`` 
+   * :issue:`` 
+
+.. admonition:: Template updates
+
+   * :issue:`3203` Added Poseidon II to RHS NAPA
+   * :issue:`3204` Added Reaction Forces CDLC compatibility for Western Sahara templates
+   * :issue:`3208` Added RHS HIDF template
```
Comment (ante185) [inline changelog.rst:46]:
```suggestion
   * :issue:`3209` Added RHS HIDF template
```
corrected PR number

File: changelog.rst [hunk 3]
```diff
@@ -3,6 +3,119 @@ Changelog
...
+   * :issue:`3254` Fixed Refugee mission timer
+   * :issue:`3256` Destroy Heli mission
+   * :issue:`3260` Artillery marker improvements
+   * :issue:`3264` Fixed lootToCrate issues with IFA gear
+   * :issue:`3266` Improved getArtilleryRanges & fixed IFA mortar case
+   * :issue:`3273` Prevented concurrent major attacks, and added a planning cost
+   * :issue:`3281` Fixed 3CB Factions baseWeapon config
+   * :issue:`3283` Fixed incorrect config array specifier for diveParams
+   * :issue:`3286` Fixed inconsistent parameter bug with createAttackForceLand
+   * :issue:`3287` Fixed WW2 factions using SAM supports
```
Comment (ante185) [inline changelog.rst:72]: ground work / new template feature, not bugfix
Goes hand in hand with the Anti-Air rework as it moved SAM out of unfair supports.

Comment (Bob-Murphy) [reply changelog.rst]: done

File: changelog.rst (hunk 2 above)
Comment (Bob-Murphy) [reply changelog.rst]: done

**Review by killerswin2 — APPROVED** [review]

**Review by ante185 — APPROVED** [review]
lgtm

**Review by Play3rgg — APPROVED** [review]
lgtm

**Pull request merged**